app/src/componentLibrary/sidebar/sidebar.scss

```diff
@@ -17,33 +17,52 @@
 .sidebar-container {
   height: 100%;
   font-family: $FONT-ROBOTO-REGULAR;
+  width: 48px;
+  transition: width 0.3s ease-out;
+  transition-delay: 0.5s;
+  position: relative;
+  z-index: 1;
+  overflow: hidden;
+
+  &:not(.open) {
+    transition-delay: 0s;
+  }
```

Comment:
Since it will be applied to `.sidebar-container` when it is not open, perhaps it is better to use `transition-delay: 0s;` by default instead of `transition-delay: 0.5s;`. And set the `transition-delay: 0.5s;` for `.open` class.
What do you think?

Reply:
It won't work as expected.

Reply:
Done.